core/classes/Avatars/AvatarSource.php

```diff
@@ -89,7 +89,7 @@ public static function getAvatarFromUserData(object $data, bool $allow_gifs = fa
             }
         }
 
-        return "https://avatars.dicebear.com/api/initials/{$data->username}.png?size={$size}";
+        return "https://api.dicebear.com/5.x/initials/svg?seed={$data->username}&size={$size}";
```

Comment:
It was switched to .png for discord reasons?`now its back to svg?

Reply:
Accidentally changed when I copied new URL from their docs 🤦🏼‍♂️